arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1082,6 +1082,9 @@
             "$ref": "#/definitions/EventHubProperties"
           }
         },
+        "routing": {
```

Comment:
Given that our RP is already live with this new property, we have a consensus now to do the following. 

- Rev the RP API version now, and once we do that, the new and old versions will start supporting the “routing” property.
- If a request comes for the old RP API version and the routing properties are set to null, we will merge the routing properties from the DB and act on the combined document. 
- If the request for the old RP API version comes with routing properties filled we will act on them as usual.

Going forward we will rev the API version for every new property introduced.